chart-parts/ingress.yaml

```diff
@@ -1,51 +1,43 @@
-{{- if .Values.services.ingress -}}
+{{- if .Values.ingress.enabled -}}
 ---
-# The cert and the key cannot be filled automatically. If we could it
-# would roughly look like the lines below. As is it is the
-# responsibility of the operator to retrieve the values of these
-# secrets from the deployed SCF and update this secret with the actual
-# values. And the responsibility of the developer to supply the proper
-# instructions to the operator.
-##
-# tls.crt: {{ default "" .Values.secrets.ROUTER_SSL_CERT     | b64enc | quote }}
-# tls.key: {{ default "" .Values.secrets.ROUTER_SSL_CERT_KEY | b64enc | quote }}
-
 apiVersion: "v1"
 kind: "Secret"
 type: kubernetes.io/tls
 metadata:
-  name: "{{ .Values.services.ingress.class }}-ingress-tls"
-  namespace: "{{ .Release.Namespace }}"
+  name: "ingress-tls"
+  namespace: {{ .Release.Namespace | quote }}
 data:
-  tls.crt: ""
-  tls.key: ""
+  tls.crt: {{ .Values.ingress.tls.crt | default "" | b64enc | quote }}
+  tls.key: {{ .Values.ingress.tls.key | default "" | b64enc | quote }}
 ---
-# This ingress specifies routing and access for the cloud controller
-# public SCF service, i.e. "api.<domain>" and other services in that
-# domain hierarchy.
-#
-# Note that the two other services (tcp routing, app ssh access) are
-# not suitable to routing via http, and not specified. IOW they are
-# TCP, which is not supported by the current ingress specification.
-# Although specific controllers may do so in a custom manner.
```

Comment:
The comment doesn't look that obsolete?

Reply:
I removed because I don't believe it belongs to the template code itself, but rather in the Wiki and official docs, as they already are.

Reply:
Added comment back.